src/Uno.UI/UI/Xaml/Shapes/Shape.wasm.cs

```diff
@@ -224,5 +213,49 @@ private UIElementCollection GetDefs()
 
 			return _defs.Defs;
 		}
+
+		private static Rect GetPathBoundingBox(Shape shape)
+		{
+			var bbox = Rect.Empty;
+
+			foreach (var child in shape.GetChildren())
+			{
+				if (child is DefsSvgElement)
+				{
+					// Defs hosts non-visual objects
+					continue;
+				}
+
+				var childRect = child.GetBBox();
```

Comment:
I think this is the native API? If I'm right this `GetBBOX` fails (especially on FF) if SVG element is not fully "loaded".